src/bitcoin-cli.cpp

```diff
@@ -867,6 +868,62 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) {
+        return;
+    }
+
+    std::string RESET = "\033[0m";
```

Comment:
Are there still issues on windows with colorized text, or can basic ANSI color support be assumed all over the board now?

Reply:
hmmm I do not own a Windows computer and cannot test if ANSI color support works. 

> (if not, you might want to put this in #ifndef WIN32, leave the strings empty if not—we do this for some python scripts)
Am I right to say you are referring to something like this? 
```
    std::string RESET = "";
    std::string GREEN = "";
    std::string BLUE = "";
    std::string YELLOW = "";
    std::string MAGENTA = "";
    std::string CYAN = "";
#ifndef WIN32
    RESET = "\033[0m";
    GREEN = "\x1B[32m";
    BLUE = "\x1B[34m";
    YELLOW = "\x1b[33m";
    MAGENTA = "\x1B[35m";
    CYAN = "\x1B[36m";
#endif
```

Reply:
Yes, for example. I think I would personally do, to make it symmetrical:
```c++
#ifndef WIN32
    const std::string RESET = "\033[0m";
    const std::string GREEN = "\x1B[32m";
    const std::string BLUE = "\x1B[34m";
    const std::string YELLOW = "\x1b[33m";
    const std::string MAGENTA = "\x1B[35m";
    const std::string CYAN = "\x1B[36m";
#else
    const std::string RESET = "";
    const std::string GREEN = "";
    const std::string BLUE = "";
    const std::string YELLOW = "";
    const std::string MAGENTA = "";
    const std::string CYAN = "";
#endif
```

Reply:
@laanwj Yeah you're right that looks better 👍. Updated the PR to add WIN32 check.